types/opentok/index.d.ts

```diff
@@ -196,10 +196,16 @@ declare module 'opentok' {
       layoutClassList: string[];
       videoType: 'camera' | 'screen' | 'custom';
     }
+
+    export interface Env {
+      apiUrl?: string;
+      uaAddendum?: string;
+      timeout?: number;
+    }
   }
 
   class OpenTok {
-    constructor(apiKey: string, apiSecret: string);
+    constructor(apiKey: string, apiSecret: string, env?: OpenTok.Env | string);
```

Comment:
You should remove the ` | string`. You can also remove `OpenTok` since it is defined a few lines above.

```ts
constructor(apiKey: string, apiSecret: string, env?: Env);
```

Reply:
Will do, I just noticed that string isn't documented. We need to keep it as `OpenTok.Env` otherwise the TS linter complains.